sdk/mixedreality/azure-mixedreality-authentication/azure/mixedreality/authentication/_utils.py

```diff
@@ -0,0 +1,73 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import base64
+import json
+import random
+
+from azure.core.credentials import AccessToken
+
+from ._generated.models import StsTokenResponseMessage
+
+
+def convert_to_access_token(token_response_message):
+    # type: (StsTokenResponseMessage) -> AccessToken
+    """
+    Converts the specified token response message to an AccessToken.
+    """
+    if not StsTokenResponseMessage:
+        raise ValueError("token_response_message can not be None")
```

Comment:
```suggestion
    if not token_response_message:
        raise ValueError("token_response_message can not be None")
```

I would also point out that these error messages may be confusing to users. The value here can't equal anything "falsey", including empty string, 0, false or None. I think it would be better to make a positive declaration of what it should be (i.e. non-empty string) rather than what it shouldn't.

If you actually want it so that the value only can't be None, but could be other falsey value, then the test should be:
```python
if token_response_message is not None:
  # stuff
```

Reply:
Yea. Might have been copy paste mistake. I'll change the message.

Reply:
Validating parameters doesn't seem "fashionable" as I don't see many other libraries doing it. So, i'm not sure what language is common to what Python users expect. I did was the `communication` library was doing.

Reply:
I've updated the message to be more "string" related.

Reply:
The most important part is the suggested code change, not the error message.

If you were to change this particular message, you could do the same in dozens of places, which is probably not necessary. @annatisch any advice here?

Reply:
Yep. I included the highlighted change you suggested.

Reply:
I also updated the message in a few other places. Let me know if there's any I missed.